lib/src/typer.dart

```diff
@@ -12,11 +12,17 @@ class TyperAnimatedText extends AnimatedText {
   /// By default it is set to 40 milliseconds.
   final Duration speed;
 
+  /// The [Curve] of the rate of change of animation over time.
+  ///
+  /// By default it is set to Curves.ease.
+  final Curve curve;
+
   TyperAnimatedText(
     String text, {
     TextAlign textAlign = TextAlign.start,
     @required TextStyle textStyle,
     this.speed = const Duration(milliseconds: 40),
+    this.curve = Curves.ease,
```

Comment:
Should add an `assert` that `curve` is _not null_ -- for the _null safety_ conversion later.

Reply:
Sure, I shall do that.